scripts/build/bundles.js

```diff
@@ -0,0 +1,109 @@
+"use strict";
+
+const path = require("path");
+
+const parsers = [
+  {
+    input: "src/language-js/parser-babylon.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-js/parser-flow.js",
+    target: "universal",
+    minify: true,
+    strict: false
+  },
+  {
+    input: "src/language-js/parser-typescript.js",
+    target: "universal",
+    minify: true
+  },
+  {
+    input: "src/language-css/parser-postcss.js",
+    target: "universal",
+    bundler: "webpack",
```

Comment:
This is the only bundle that uses webpack. Why doesn't it work with rollup? If we can get it working with rollup, then we won't need to support webpack, which simplifies the build script a lot.

Reply:
Oh I forgot to bring the comment from the old `build/build.js`. postcss has a dependency cycle and doesn't work with rollup. Will add a comment